### ISSUE TITLE
Travis coverage for Dockerfile

### ISSUE THREAD

**Issue by deed02392** [jfurrow/flood#592]
Just a suggestion - should we implement Travis CI coverage for the Dockerfile from this repo? We can verify that changes don't affect the Docker building and functionality. I have not got a lot of experience with Travis beyond having it do a build, but I believe it is possible to also test the functionality of the container too.

**Comment by noraj1337** [comment]
I agree but Travis is a solution, what we need is a CI for the Dockerfile, maybe in this case another CI will do a better Job.

**Comment by deed02392** [comment]
We could just add test scripts to the existing `.travis.yml`.

Something like:

`install:
  - docker build -t flood .
  - docker start -d -p 127.0.0.1:80:80 --name flood flood
script:
  - docker ps | grep -q blog
`

**Comment by frebib** [comment]
I'd like to add that the tests above should test that the web interface loads, possibly with curl? Also the port of the container need not be exposed, but instead the ip of the container used instead to communicate with it directly

**Comment by noraj** [comment]
@frebib Good idea, build a `.travis.yml` file and commit it to a test branch ;)

**Comment by frebib** [comment]
I can't say I've ever used Travis, but sure. When I get 5 minutes after my exams are done I'll give it a try
